packages/checkout/shipping/shipping-rates-control/shipping-rates-control.js

```diff
@@ -14,9 +14,25 @@ import {
 /**
  * Internal dependencies
  */
-import Packages from './packages';
+import Package from './package';
+import ExperimentalOrderShippingPackages from '../../order-shipping-packages';
 import './style.scss';
 
+/**
+ * @typedef {import('react')} React
+ */
+
+/**
+ * Renders the shipping rates control element.
+ *
+ * @param {Object} props Incoming props.
+ * @param {Array} props.shippingRates Array of packages containing shipping rates.
+ * @param {boolean} props.shippingRatesLoading True when rates are being loaded.
+ * @param {string} props.className Class name for package rates.
+ * @param {boolean} props.collapsibleWhenMultiple If true, when multiple packages are rendered they can be toggled open and closed.
+ * @param {React.ReactElement} props.noResultsMessage Rendered when there are no packages.
+ * @param {Function} props.renderOption Function to render a shipping rate.
+ */
 const ShippingRatesControl = ( {
```

Comment:
In a follow up, this component, with `TotalsShipping` and `ShippingRateSelector` need to be moved back to our base components folder, they're too coupled with our base implementation to be exported, if anything, we should only export `Package`.